test/integration/targets/filters/runme.sh

```diff
@@ -7,6 +7,6 @@ source virtualenv.sh
 # Requirements have to be installed prior to running ansible-playbook
 # because plugins and requirements are loaded before the task runs
 
-pip install jmespath netaddr
+pip install jmespath netaddr jinja2 cryptography pyyaml
```

Comment:
wouldn't Ansible install itself take care of those dependencies?

Reply:
Removed those. I have something weird going on in my normal setup of pyenv + virtualenv that breaks with all the recent virtualenv.sh test changes. :-(